rulebook.tex

```diff
@@ -1738,7 +1764,42 @@ \subsection{Task Fulfillment and Scoring}
         \\
         Retrieve from SS
         & A workpiece has been requested from storage and is ready for retrieval
-        & $-10$
+        & $-5$
+        \\
+        Store at SS
+        & A workpiece is stored into the SS
+        & $-5$
+        \\
+        Relocate at SS
+        & A storage/retrieval request to an inaccessible slot requires
+        relocation of a blocked workpiece
+        & $0$
+        \\
+        Upkeep at SS
+        & A workpiece stored at time $t$ causes costs $f(x)$ at
+        ${t+x\cdot\SI{60}{\second}}$:
+        \begin{tikzpicture}[baseline={(current bounding box.north)}]
+          \node[align=left] (conditions) {
+           if $x < 3$\\
+           if $3\leq x \leq 7$\\
+           else
+            };
```

Comment:
> I find the cost scheme rather complicated. What's the motivation for the upkeep cost? Do we expect this to actually affect the game? If a team decides to use the storage station, will they actually consider the storage time and the related upkeep time when making the decision?

The Motivation behind that was to reflect the costs associated with occupying real storage space which typically also accumulates costs over time in the form of rent, logistics maintenance etc. In the TC meeting @chde stressed that this is an important aspect when modeling storage costs. I agree that with the proposed values it is unlikely that decisions depend on considered upkeep costs. I thought it would make sense to introduce the type of costs already as a concept such that we can adapt the values once teams figure out appropriate applications for the SS. With the current proposal this slightly discourages people to store products without retrieving them later (which is something that you probably should not do in most production domains).

That said i am also not opposed to keeping the first set of rules even simpler by sticking to storage and retrieval costs only and leaving a total cost cap and upkeep cost out of the equation for now.

Reply:
> This would be easier to understand if it was the total cost, not the cost increment per minute.

Okay, here is a different formulation: https://github.com/robocup-logistics/rcll-rulebook/pull/40/commits/abb236c1b02af67e15f6e6d69adead5d33b720f7.

Reply:
> That said i am also not opposed to keeping the first set of rules even simpler by sticking to storage and retrieval costs only and leaving a total cost cap and upkeep cost out of the equation for now.

Such a simplification could look like this: https://github.com/robocup-logistics/rcll-rulebook/pull/40/commits/022ee13f44dee59036fd19a0d837cdcaf7238aab